src/videos/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.8-slim
```

Comment:
Change to image in public ECR repo to avoid Docker Hub API throttling issues.

```
FROM public.ecr.aws/s5z3t2n9/python:3.8-slim
```

Reply:
This change is from after the fork. It should be resolved with a merge from master.